verification/io/io-spec.gobra

```diff
@@ -92,79 +109,88 @@ pure func dp3s_iospec_bio3s_enter_guard(s IO_dp3s_state_local, t Place, v IO_val
 				v.IO_Internal_val1_4)
 }
 
-pred dp3s_iospec_bio3s_enter(s IO_dp3s_state_local, t Place) {
+pred (dp DataPlaneSpec) dp3s_iospec_bio3s_enter(s IO_dp3s_state_local, t Place) {
 	// TODO: we may need more triggering terms here
-	forall v IO_val :: { dp3s_iospec_bio3s_enter_guard(s, t, v) } (v.isIO_Internal_val1 ==>
-			(dp3s_iospec_bio3s_enter_guard(s, t, v) ==>
-				(CBio_IN_bio3s_enter(t, v) &&
-				dp3s_iospec_ordered(
-					dp3s_add_obuf(s, v.IO_Internal_val1_4, v.IO_Internal_val1_3),
-					CBio_IN_bio3s_enter_T(s, t, v)))))
+	forall v IO_val :: { dp.dp3s_iospec_bio3s_enter_guard(s, t, v) } (
+		match v {
+			case IO_Internal_val1{_, _, ?newpkt, ?nextif} : (dp.dp3s_iospec_bio3s_enter_guard(s, t, v) ==>
+				(dp.CBio_IN_bio3s_enter(t, v) &&
```

Comment:
to make it a bit easier to read, I would always add a line break after `:` in the line with the `case` clause (everywhere)

Reply:
What about cases like this `case none[IO_seg2]: false`? Do you still want a new line? Also, I assume that you want a line break for the `default` case?

Reply:
yeah, I would always put a line break to be consistent.